freqtrade/exchange/binance.py

```diff
@@ -163,12 +182,14 @@ def get_max_leverage(self, pair: Optional[str], nominal_value: Optional[float])
         """
         if pair not in self._leverage_brackets:
             return 1.0
+        if (pair is None or nominal_value is None):
+            raise OperationalException(
+                "binance.get_max_leverage requires parameters pair and nominal_value")
         pair_brackets = self._leverage_brackets[pair]
-        max_lev = 1.0
-        for [min_amount, margin_req] in pair_brackets:
-            if nominal_value >= min_amount:
-                max_lev = 1/margin_req
-        return max_lev
+        for [notional_floor, mm_ratio, _] in reversed(pair_brackets):
+            if nominal_value >= notional_floor:
+                return 1/mm_ratio
```

Comment:
Can we rely on the fact that mm_ratio will never be 0.0? Otherwise it'll cause a 0 Division exception.

Reply:
> Can we rely on the fact that mm_ratio will never be 0.0? Otherwise it'll cause a 0 Division exception.

It looks like that, there isn't 1 leverage_bracket with 0 as the margin ratio